maven-model/src/test/resources/xml/pom.xml

```diff
@@ -37,12 +37,12 @@ under the License.
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-api-model</artifactId>
-      <version>4.0.0-alpha-1-SNAPSHOT</version>
+      <version>${project.version}</version>
```

Comment:
I'd rather have all the versions in this file set to `dummy-version` with no filtering... see comment above.